testsuite/integration/microprofile-tck/openapi/pom.xml

```diff
@@ -172,7 +172,7 @@
             <id>bootablejar.ee9.profile</id>
             <activation>
                 <property>
-                    <name>ts.bootable.ee9</name>
+                    <name>DISABLED.ts.bootable.ee9</name>
```

Comment:
Why disable these? We still want to test WFP, which I expect will start having greater differences from std WF soon.

We disabled some of these during the big bang but that seemed more of a case of getting rid of one moving part while everything was in flux.

Reply:
Same question applies re the 3rd commit.

Reply:
Good point.  These changes had been in my branch since before the big-bang, but they are no longer necessary.  I'll update accordingly.